assets/blocks/view-certificate-block/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerBlockVariation } from '@wordpress/blocks';
+
+const buttonVariationAttributes = {
+	url: '/course/sensei-view-certificate',
+	title: __( 'View Certificate', 'sensei-certificates' ),
+	text: __( 'View Certificate', 'sensei-certificates' ),
+	linkTarget: '_blank'
+};
+
+registerBlockVariation( 'core/button', {
+	name: 'view-certificate-button',
+	attributes: buttonVariationAttributes
+} );
+
+registerBlockVariation( 'core/buttons', {
+	name: 'sensei-lms/view-certificate-buttons',
```

Comment:
I think in this case we should use `sensei-certificates/` as prefix to this plugin.

Reply:
Fixed in cb0aa85bca5dbf5c231cddcbe16ab6cd7b0c84ad